params/protocol_params.go

```diff
@@ -48,11 +48,11 @@ const (
 	CallStipend           uint64 = 2300  // Free gas given at beginning of call.
 	ETXGas                uint64 = 21000 // Per ETX generated by opETX or normal cross-chain transfer.
 	//  The etx fractions  should be based on the current  expansion number
-	ETXRegionMaxFraction int    = 1   // The maximum fraction of transactions for cross-region ETXs
-	ETXPrimeMaxFraction  int    = 1   // The maximum fraction of transactions for cross-prime ETXs
-	ETXRLimitMin         int    = 10  // Minimum possible cross-region ETX limit
-	ETXPLimitMin         int    = 10  // Minimum possible cross-prime ETX limit
-	EtxExpirationAge     uint64 = 100 // Number of blocks an ETX may wait for inclusion at the destination
+	ETXRegionMaxFraction int    = 1          // The maximum fraction of transactions for cross-region ETXs
+	ETXPrimeMaxFraction  int    = 1          // The maximum fraction of transactions for cross-prime ETXs
+	ETXRLimitMin         int    = 10 * 21000 // Minimum possible cross-region ETX gas limit
+	ETXPLimitMin         int    = 10 * 21000 // Minimum possible cross-prime ETX gas limit
```

Comment:
Should be able to use ETXGas here as well

Reply:
Done

Reply:
Btw, ETXGas is how much is used in the origin, TxGas is (generally) how much is used in the destination. The limit is set for gas expected to be used in the destination.